specification/suppressions.yaml

```diff
@@ -54,3 +54,977 @@
     - sphere/resource-manager/Microsoft.AzureSphere/preview/2022-09-01-preview/azuresphere.json
     - splitio/resource-manager/SplitIO.Experimentation/preview/2024-03-01-preview/splitio.json
     - translation/data-plane/Azure.AI.DocumentTranslation/preview/2023-11-01-preview/openapi.json
+- tool: TypeSpecValidation
```

Comment:
I think suppressions should be organized as one group per spec.  Unrelated specs shouldn't share suppressions.